router/src/extract_routes.rs

```diff
@@ -11,7 +11,7 @@ use std::{
 
 /// Context to contain all possible routes.
 #[derive(Clone, Default, Debug)]
-pub struct PossibleBranchContext(pub(crate) Rc<RefCell<Vec<Branch>>>);
+pub struct PossibleBranchContext(pub Rc<RefCell<Vec<Branch>>>);
```

Comment:
TODO: I wasn't happy with making this public.  It was a quick hack for `test_unique_route_ids()`.

I think I can revert this if I move the tests from a standalone `testing/foo.rs` file to a test module. Happy to follow your suggestions here.